pyproject.toml

```diff
@@ -0,0 +1,72 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=77.0.0"]
+
+[project]
+name = "vip_hci"
+version = "1.6.6"
+description = "Package for astronomical high-contrast image processing."
+readme = "README.rst"
+license = "MIT"
+license-files = ["LICENSE"]
+authors = [
+  { name = "Carlos Alberto Gomez Gonzalez" },
+  { name = "Valentin Christiaens", email = "valentin.christiaens@uliege.be" },
+]
+requires-python = ">=3.8"
+classifiers = [
+  "Intended Audience :: Science/Research",
+  "Natural Language :: English",
+  "Operating System :: MacOS :: MacOS X",
+  "Operating System :: POSIX :: Linux",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Topic :: Scientific/Engineering :: Astronomy",
+]
+urls.Homepage = "https://github.com/vortex-exoplanet/VIP"
+dependencies = [
+    "numpy",
+    "scipy",
+    "astropy",
+    "photutils",
+    "scikit-learn",
+    "scikit-image",
+    "emcee",
+    "nestle",
+    "corner",
+    "pandas",
+    "matplotlib>=3.7.0",
+    "psutil",
+    "pyprind",
+    "munch",
+    "hciplot>=0.2.4",
+    "dataclass_builder",
+]
+
+[project.optional-dependencies]
```

Comment:
note: this is strictly equivalent to what used to be defined dynamically in `setup.py`, however I believe there's now a better approach with recent standards, using a [PEP 735 dependency group](https://peps.python.org/pep-0735/), but that would require dropping support for Python 3.8 (end of life), because no version of `pip` supports both.